Desktop/data/columnsmodel.cpp

```diff
@@ -214,8 +226,8 @@ bool ColumnsModel::absorbInfo(VariableInfo::InfoType info, const QString &colNam
 		switch(info)
 		{
 		default:										return	false;
-		case VariableInfo::DataSetValue:				return	QTransposeProxyModel::setData(qValIndex, value, int(DataSetPackage::specialRoles::value));
-		case VariableInfo::DataSetValues:				return	QTransposeProxyModel::setData(qColIndex, value,	int(DataSetPackage::specialRoles::valuesStrList));
+		case VariableInfo::DataSetValue:				return	QAbstractTableModel::setData(qValIndex, value, int(DataSetPackage::specialRoles::value));
```

Comment:
The qValIndex is set as if it was still the transposed proxy model. So that won't work

Reply:
If you have the fix you can push it ^^